services-geocoding/src/main/java/com/mapbox/api/geocoding/v5/MapboxGeocoding.java

```diff
@@ -560,6 +564,25 @@ public abstract Builder reverseMode(
      */
     public abstract Builder fuzzyMatch(Boolean fuzzyMatch);
 
+    /**
+     * Specify whether to request additional metadata about the recommended navigation destination
+     * corresponding to the feature (true) or not (false, default).
+     * Only applicable for address features.
+     *
+     * For example, if routing=true the response could include data about a point
+     * on the road the feature fronts.
+     * Response features may include an array containing one or more routable points.
+     * Routable points cannot always be determined. Consuming applications should fall back to
+     * using the feature's normal geometry for routing if a separate routable point
+     * is not returned.
+     *
+     * @param routing optionally set whether to request
+     *                additional metadata about the recommended navigation destination
+     * @return this builder for chaining options together
+     * @since 6.10.0
```

Comment:
```suggestion
```

Reply:
Thanks, I've removed `@since` note everywhere in this PR